analytic_engine/src/sst/parquet/encoding.rs

```diff
@@ -175,8 +197,41 @@ pub enum Error {
 define_result!(Error);
 
 pub const META_KEY: &str = "meta";
+pub const META_PATH_KEY: &str = "meta_path";
+pub const META_PATH_VERSION_KEY: &str = "meta_path_version";
+pub const META_PATH_VERSION_V1: &str = "meta_path_version_v1";
+pub const META_PATH_VERSION_V2: &str = "meta_path_version_v2";
```

Comment:
```
META_VERSION :&str = "2";
```
1 is not used, when `META_PATH_VERSION_KEY` is not found in KV, then it will be version 1.